grimoire_elk/enriched/enrich.py

```
@@ -233,7 +237,7 @@ def __compare_projects_map(self, db, json):
                     db_projects.append(project)
                 if project not in json:
                     logger.error("Project not found in JSON ", project)
-                    raise
+                    raise NotFoundError("Project not found in JSON " + project)
```

Thanks for improving this code also in this PR.  Much better to raise a specific exception here.